integrations/purchase/nag-categories.js

```diff
@@ -33,371 +33,119 @@ import {
   PURCHASE_CATEGORY_TRANSPORT_ROAD,
   PURCHASE_CATEGORY_ALCOHOL,
   PURCHASE_CATEGORY_OTHER_ENERGY_HEATING_COOLING,
-  ACTIVITY_TYPE_TRANSPORTATION,
-  TRANSPORTATION_MODE_CAR,
+  PURCHASE_CATEGORY_ELECTRICITY,
 } from '../../definitions';
 
+const purchaseActivity = purchaseType =>
```

Comment:
The helper here is nice for DRYness, but I would remove the ternary inside it so `purchaseActivity` always returns a purchase.

Reply:
If we decide that activityType is always purchase, wouldn't it then make more sense to get rid of this object all together and just use the purchaseType directly in the integration?

Example:
```js
export const NAG_CATEGORY = {
  'ATM & Checks': PURCHASE_CATEGORY_FINANCIAL_SERVICES,
  'Advisors & Services': PURCHASE_CATEGORY_OTHER_SERVICES,
};
```

Reply:
Yes indeed I was thinking about that as well

Reply:
Nice change! 💪